File: 242-valid-anagram/242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s1, string s2) {
          if(s1.Length != s2.Length)
               return false;

            int[] charCount = new int[26];
             
           for(int i=0;i < s1.Length;i++)
           {
              charCount[s1[i] - 'a']++;
              charCount[s2[i] - 'a']--;
           }

           foreach(var idx in charCount)
           {
                if(idx > 0)
                  return false;
           }
        
         return true;
    }
}

File: 242-valid-anagram/242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        
        int[] alphabets = new int[26];
        
        
        if(s.length() != t.length())
            return false;
        
        for(int i=0;i < s.length();i++)
        {
            alphabets[s.charAt(i) - 'a']++;
            alphabets[t.charAt(i) - 'a']--;
        }
        
        for(int n : alphabets)
        {
            if(n != 0)
                return false;
        }
        
        return true;
        
        
    }
}